src/components/Page/FolderPage.js

```diff
@@ -0,0 +1,62 @@
+import AddLinkForm from "../AddLinkForm";
+import '../../styles/Folder.css';
+import SearchBar from "../SearchBar";
+import NoLink from "../NoLink";
+import FolderLists from "../FolderLists";
+import Card from "../Card";
+import FolderTitle from "../FolderTitle";
+import {  useFolderPageFolder } from "../../hooks/folderPageHooks";
+import { useEffect, useState } from "react";
+import { ALL_FOLDER, FOLDER_PAGE_API_URL } from "../../constants/constants";
+import axios from "axios";
+
+const linkURL = (forderId) => {
+  if(forderId === 0) return `${FOLDER_PAGE_API_URL}/1/links`;
+  return `${FOLDER_PAGE_API_URL}/1/links?folderId=${forderId}`;
+}
+
+const FolderPage = () => {
+  const [activeFolder, setActiveFolder] = useState(0);
+  const folders =[ALL_FOLDER, ...useFolderPageFolder()];
+  const [links, setLinks] = useState()
+
+  const getLinkData = async (URL) => {
+    try{
+      const response = await axios.get(URL);
+      setLinks(response.data.data);
+    } catch(error){
+      alert(error)
+    }
+  };
+  
+  useEffect(() => {
+    const url = linkURL(activeFolder);
+    getLinkData(url);
+  },[activeFolder])
+
+  if(!folders) return;
+
+  const handleActiveButton = (id) => {
+    setActiveFolder(id);
+  }
+
+  return(
+    <main className="contents folder-link--main">
+      <AddLinkForm />
+      <SearchBar/>
+      {!links && <NoLink />}
+      {links &&
+        <>
+          <FolderLists folders={folders} activeFolder={activeFolder} onClick={handleActiveButton}/>
+          <FolderTitle folders={folders} activeFolder={activeFolder}/>
+          {links.length === 0 && <NoLink />}
+          <section className="card--section">
+            {links?.map((link) => <Card key={link.id} link={link} />)}
+          </section>
```

Comment:
받아온 데이터가 없는 경우에 처리가 잘 되었네요 👍 

지금은 NoLink랑 `<section className="card--section">`이 동시에 그려질수도 있을 텐데, 그럴 필요도 없고 `card--section`이 크기를 갖게 되면 레이아웃이 의도와 달라질 수 있겠네요

```suggestion
          {links.length === 0 
          	? <NoLink />
          	: (
	  	  <section className="card--section">
	            {links?.map((link) => <Card key={link.id} link={link} />)}
	          </section>
          	)
        }
```